app/views/spree/taxons/_sorting_filters.html.erb

```diff
@@ -27,7 +28,14 @@
       <button type="button" class=" ">Z to A</button>
     <%end%>
   </div>
-  <button type="button" class="">Best Sellers</button>
+
+  <div class="product-sort-js" data-filter-type="bestsellers-filter" >
+    <% if params[:sorting].present? && params[:sorting] == "bestsellers_filter" %>
+       <button type="button" class="active">Best Sellers</button>
```

Comment:
If & else code is almost same except Active class. Please make new variable for klass and call dynamically